### PR TITLE
allow refType to be defined in @prop (issue #146)

### PR DESCRIPTION
**Pull request by mattjennings**

### CODE REVIEW

File: src/prop.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ export interface BasePropOptions {
 
 export interface PropOptions extends BasePropOptions {
   ref?: any;
+  refType?: any;
```
Comment (Ben305) [inline src/prop.ts]: Is ˋanyˋ really correct here? What about ˋNumber | String | Bufferˋ?

Comment (mattjennings) [reply src/prop.ts]: No, you're right. It uses a mongoose.Schema.Types object, but I'll find the correct way to type it.

**Comment by mattjennings** [comment]
I realized I had other commits from my fork coming through on this by accident, I'll create a proper pull request soon.

**Comment by Ben305** [comment]
Great, I love clean, focused PRs!

**Pull request closed without merging**